lib/tx-result-adapters.ts

```diff
@@ -157,27 +193,23 @@ export class LbdTxMessageAdapterV1 implements TxResultAdapter<RawTransactionResu
 
 export class LbdTxEventsAdapterV1 implements TxResultAdapter<RawTransactionResult, Set<TransactionEvent>> {
   txEVentConverter: LbdTxEventConverterV1;
-
   constructor() {
     this.txEVentConverter = new LbdTxEventConverterV1();
   }
-
   adapt(input: RawTransactionResult): Set<TransactionEvent> {
     let logs = input.logs;
 
     if (input.code !== 0 || _(logs).isEmpty()) {
-      // TODO create const empty
-      console.warn("tx-result logs is empty");
-      return new Set();
+      return EMPTY_TX_EVENTS;
     } else {
       let events = _.flatMap(logs, log => {
         let msgIndex = log.msgIndex;
         let eventType = RawMessageEventKeyTypeUtil.convertToEventType(input.tx.value.msg[msgIndex].type);
-        let event = RawTransactionLogUtil.findEvent(log, eventType);
```

Comment:
This is just curious.
Which is correct, using or not using a semicolon?

Reply:
Well it's kind of decision issue, which means it's up to our decision. But I want to use it even though I also missed sometimes. :)